projectile.el

```diff
@@ -950,7 +951,11 @@ Files are returned as relative paths to the project root."
   :group 'projectile
   :type 'string)
 
-(defcustom projectile-fossil-command "fossil ls | tr '\\n' '\\0'"
+(defcustom projectile-fossil-command (concat "fossil ls | "
+                                             (when (string-equal system-type
```

Comment:
This `when` will return `nil` on non-windows systems. Don't recall if `concat` handles `nil` or not.